src/Libraries/GeometryColor/ImportSatWithUnits.cs

```diff
@@ -0,0 +1,68 @@
+﻿using DynamoUnits;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.IO;
+using Autodesk.DesignScript.Runtime;
+
+//TODO at a major host release verion - 2024 for example, rename this assembly to something like
+//ZeroTouchGeometryNodes and use type forward attribute to retain API compat.
+
+//This namespace and class name are a bit confusing - but this is done so that the Nodes show up in the UI as Geometry.NodeName.
+namespace Autodesk.DesignScript.Geometry.ImportHelpers
```

Comment:
I'm not thrilled with this namespace and class name, but it works so that the Node appears to be in the Geometry category and class when in the UI and it's not brutal when using from c# or a CBN, though there could be issues disambiguating `Autodesk.DesignScript.Geometry.Geometry` from `Autodesk.DesignScript.Geometry.ImportHelpers.Geometry` ... A simpler solution is to just use a different class name here but I was not sure it was acceptable to have the Import nodes change their class name, even if they are still shown in the Geometry category in the library.

If that is acceptable, than I think it's a better, simpler solution for DS/C#/python users and doesn't confuse things too bad for node UI users much.

@aparajit-pratap let me know what you think.

Reply:
@mjkkirschner this is extra work but how about exposing these nodes as nodemodel nodes like we do for `ExportWithUnits` - https://github.com/DynamoDS/Dynamo/blob/4819d581c7386adafc9558a5b8a74ef502873aaf/src/Libraries/GeometryUI/ExportWithUnits.cs#L15. This way we can make them seamlessly look like they're part of the Geometry class and category in the library.